packages/api/src/modules/providers/apiAsso/apiAsso.service.ts

```diff
@@ -60,18 +61,19 @@ export class ApiAssoService implements AssociationsProvider, EtablissementProvid
         const rnaStructure = await this.sendRequest<RnaStructureDto>(`/api/rna/${rna}`);
 
         if (!rnaStructure) return null;
-        if (!rnaStructure.identite?.date_modif_rna) return null; // sometimes an empty shell object if given by the api
+        if (hasEmptyProperties(rnaStructure.identite) || !rnaStructure.identite.date_modif_rna) return null; // sometimes an empty shell object if given by the api
```

Comment:
```suggestion
        if (hasEmptyProperties(rnaStructure.identite)) return null; // sometimes an empty shell object if given by the api
```

le 2e test est devenu redondant non ? (ça concerne d'autres lignes de ce fichier aussi)

Reply:
non c'est cas 1) j'ai un object { id: null, date_modif_rna: null } et l'autre cas c'est bien vérifié que si on a un object avec des propriétés remplies on ait bien date_modif_rna de défini (on pourrait avoir { id: "monID", date_modif_rna: undefined }

Reply:
Be si je comprends bien, non : dans le cas `{ id: "monID", date_modif_rna: undefined }` alors `hasEmptyProperties` devrait renvoyer `true` non ?

Reply:
2 cas possible:

`structure.identite = null` 

ou
 
 ```js
structure.identite = { 
 date_modif_siren: null,
id_siren: null,
...
}
```

Reply:
mais j'ai fixé ce bug selon le commentaire de @VSerain et je sais pas quel le format retourné réel en cas de bug

Reply:
`hasEmptyProperties` retourne `true` dès qu'**une** de ses propriétés est `falsy` non ?

Reply:
non c'est un `every` donc il retourne `true` si toutes les prop sont `falsy`

Reply:
Les TUs sur `ObjectHelper` expliquent le comportement normalement

Reply:
ok ok mal lu my bad. Ya un intitulé de test qui n'est pas bon mais c'est pas ça qui m'a induit en erreur (ie c'est ma fatigue)